src/app.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from "react";
+
+import './styles/app.css';
+
+const menuItems = [
+    {name: "Процессоры"},
+    {name: "Видеокарты"},
+    {name: "Материнские платы"}, 
+    {name: "ОЗУ"},
+    {name: "Устройства"}
+    ];
+    
+class Menu extends React.Component {
```

Comment:
it's app.js, so class name also should be App

App should contain 2 components - Navigation and Switch with Routes